docs/configuration/configuration.md

```diff
@@ -376,6 +376,11 @@ metric_relabel_configs:
 # 0 means no limit. This is an experimental feature, this behaviour could
 # change in the future.
 [ target_limit: <int> | default = 0 ]
+
+# Limit on total number of positive and negative buckets allowed in a native
+# histogram. If this is exceeded, the entire scrape will be treated as failed.
+# 0 means no limit.
+[ sample_limit: <int> | default = 0 ]
```

Comment:
This should be `bucket_limit` (as defined in config/config.go)

On a more general note: Maybe it should be named `native_histogram_bucket_limit`. It's a bit long, but "bucket" is a quite general term. Or to make it a bit shorter `histogram_bucket_limit` (which has the potential of getting confused with classic histograms, but that's less of a problem). Your pick…

And maybe emphasize in the doc comment that this is for each single histogram, not for all histograms together in the scrape. (It's already correctly described, but could be easily missed.) Maybe "allowed in a native histogram" → "allowed in a single native histogram"